testsuite/unittests/test_gif_cond_exp_multisynapse.sli

```diff
@@ -0,0 +1,115 @@
+/*
+ *  test_gif_cond_exp_multisynapse.sli
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+ /* BeginDocumentation
+Name: testsuite::test_gif_cond_exp_multisynapse - sli script for test of gif_cond_exp_multisynapse with external DC current and spike generators
+
+Synopsis: (test_gif_cond_exp_multisynapse) run
+
+
+Description:
+
+ This testscript checks the correctness of gif_psc_exp neuron model. It injects external DC current and also three spikes to two neurons and measures their spike times. The spike times should match the expected values.
+
+Author:  March 2016, Setareh
+SeeAlso: gif_cond_exp_multisynapse, testsuite::test_gif_cond_exp
+*/
+
+% This test should only run if we have GSL
+statusdict/have_gsl :: not {statusdict/exitcodes/success :: quit_i} if
+
+(unittest) run
+/unittest using
+
+0.1 /h Set
+
+ResetKernel
+
+0 << 
+      /local_num_threads 1 
+      /resolution h
+      /grng_seed 0
+      /rng_seeds [0]
+    >> SetStatus
+
+% neuron1 should fire spikes at 31.4, 68.5, 106.6 and 144.1 ms
+314  /t1_neuron1  Set 
+685  /t2_neuron1  Set
+1066 /t3_neuron1  Set
+1441 /t4_neuron1  Set
+ 
+% neuron2 should fire spikes at 16.8 and 70.1 ms
+168  /t1_neuron2  Set 
+701  /t2_neuron2  Set
+
+
+/gif_cond_exp_multisynapse Create /neuron1 Set % neuron1 has default parameters
+neuron1  << /taus_syn [8.0] >> SetStatus
+
+/gif_cond_exp_multisynapse Create /neuron2 Set
+neuron2  << /C_m 40.0 /Delta_V 0.2 /taus_syn [8.0] /tau_sfa [120.0 10.0] /q_sfa [10.0 25.0] /tau_stc [10.0] /q_stc [20.0] >> SetStatus
+
```

Comment:
Same general comments apply as for `test_gif_cond_exp`.  For the multisynapse types, you should also have test cases with multiple synapse time constants to make sure that that actually works.